### PR TITLE
Adapt pricing info

### PR DESCRIPTION
**Pull request by Kirastina**
As suggested by Max

### CODE REVIEW

File: site/content/english/pricing.md [hunk 1]
```diff
@@ -684,27 +684,24 @@ with the highest data protection standards in Switzerland.
...
         <td>—</td>
         <td>
           <span class="hide-for-desktop">{{< pricing-info filled=true >}}</span>
           <span class="hide-for-mobile">CHF 1’900 (one-time fee)</span>
         </td>
-        <td>
-          <span class="hide-for-desktop">{{< pricing-info >}}</span>
-          <span class="hide-for-mobile">CHF 1’900, (one-time fee)</span>
         </td>
+        <td>{{< pricing-check >}}</td>
```
Comment (ronioeschger) [inline site/content/english/pricing.md:696]: This is odd in two ways:
1. You remove the lines directly above the ones which are further above also handle that topic. Please check this out.
2. In french and german below you do it differently.

What is the actual goal of this PR @Kirastina?

**Review by ronioeschger — CHANGES_REQUESTED** [review]

File: site/content/english/pricing.md (hunk 1 above)
Comment (fursinger) [reply site/content/english/pricing.md:696]: I don't understand the 1st point. 

I agree with @ronioeschger on the 2nd point. I'll add the suggestion to the lines affected.

File: site/content/french/pricing.md [hunk 2]
```diff
@@ -694,8 +693,8 @@ Skribble permet de signer vos contrats en un clin d’œil – Validité mondial
           <span class="hide-for-mobile">CHF 1’900.- (montant unique)</span>
         </td>
         <td>
```
Comment (fursinger) [inline site/content/french/pricing.md:695]:
```suggestion
        <td>{{< pricing-check >}}</td>
```

File: site/content/french/pricing.md [hunk 3]
```diff
@@ -694,8 +693,8 @@ Skribble permet de signer vos contrats en un clin d’œil – Validité mondial
...
-          <span class="hide-for-desktop">{{< pricing-info >}}</span>
-          <span class="hide-for-mobile">CHF 1’900.- (montant unique)</span>
+          <span class="hide-for-desktop">{{< pricing-check >}}</span>
+          <span class="hide-for-mobile">{{< pricing-check >}}</span>
```
Comment (fursinger) [inline site/content/french/pricing.md:697]:
```suggestion
```

File: site/content/german/pricing.md [hunk 4]
```diff
@@ -681,18 +681,16 @@ Mit Skribble signieren Sie Ihre Verträge innert Sekunden – global rechtsgült
...
-            <span>{{< pricing-info >}} <strong>E</strong>: CHF 1’900.–, einmalig</span>
+            <span>{{< pricing-info filled=true >}} CHF 1’900.–, einmalig</span>
           </div>
         </td>
         <td>—</td>
         <td>
           <span class="hide-for-desktop">{{< pricing-info filled=true >}}</span>
           <span class="hide-for-mobile">CHF 1’900.–, einmalig</span>
-        </td>
         <td>
```
Comment (fursinger) [inline site/content/german/pricing.md:691]:
```suggestion
        <td>{{< pricing-check >}}</td>
```

File: site/content/german/pricing.md [hunk 5]
```diff
@@ -681,18 +681,16 @@ Mit Skribble signieren Sie Ihre Verträge innert Sekunden – global rechtsgült
...
-          <span class="hide-for-desktop">{{< pricing-info >}}</span>
-          <span class="hide-for-mobile">CHF 1’900.–, einmalig</span>
+          <span class="hide-for-desktop">{{< pricing-check >}}</span>
+          <span class="hide-for-mobile">{{< pricing-check >}}</span>
```
Comment (fursinger) [inline site/content/german/pricing.md:693]:
```suggestion
```

**Review by ronioeschger — APPROVED** [review]

**Pull request merged**